### PR TITLE
Bootstrap adaptions

### PR DESCRIPTION
**Pull request by FinalAngel**
- align layout with other essential addons
- cleanup markup
- validation

### CODE REVIEW

File: aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/includes/job.html [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+    {% block jobs_meta %}
+        <div class="aldryn-jobs-meta">
+            <span class="fa fa-book"></span>
+            <a href="{{ job_offer.category.get_absolute_url }}">{{ job_offer.category }}</a>
+            {% if job_offer.publication_start %}
+                &nbsp;
+                <span class="fa fa-clock-o"></span> {{ job_offer.publication_start|date  }}
+            {% endif %}
+        </div>
+        <hr />
```
Comment (dhorelik) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/includes/job.html]: `<hr>`

File: aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html [hunk 2]
```diff
@@ -1,39 +1,20 @@
...
 {% endblock %}
 
-
 {% block jobs_footer %}
     <div class="aldryn-jobs-pager">
         <ul class="pager">
             <li>
                 <span class="pager-back">
-                    – <a href="{{ job_offer.get_absolute_url }}">
+                    – <a href="{% namespace_url 'job-offer-list' %}">
```
Comment (dhorelik) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html]: maybe `&mdash;` here and below?

Comment (FinalAngel) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html]: this is currently the case for all addons

Comment (dhorelik) [inline aldryn_jobs/boilerplates/bootstrap3/templates/aldryn_jobs/jobs_detail.html]: ok then

**Comment by dhorelik** [comment]
LGTM

**Pull request merged**